en/docs/configurations.md

```diff
@@ -0,0 +1,24 @@
+# Configurations
+
+The extension offers a number of settings. These can be configured by modifying your [User or Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings), or by modifying the VSCode's ` settings.json ` file.
+
+| ID                                    | Description                                               |
+|---------------------------------------|-----------------------------------------------------------|
+| ballerina.codeLens.all.enabled        | Enable code lenses that appear in the editor view.           |
+| ballerina.enableConfigurableEditor    | Enable configurable editor. This will open a  form to set values for [configurable variables](https://ballerina.io/learn/by-example/configurable-variables/) in the code when you [Run](../docs/build-and-run/build-and-run.md) the Ballerina code. |
+| ballerina.enableNotebookDebug         | Enable debug feature in Ballerina notebook files (`.balnotebook`). |
+| ballerina.enablePerformanceForecast   | Enable performance forecaster. This will show you the forecasted latency and other performance values for Ballerina services in the low-code diagram.     |
+| ballerina.enableSemanticHighlighting  | Enable [semantic code highlighting](https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide).               |
+| ballerina.enableTelemetry             | Enable telemetry logging. This will send log data to plugin developers. This will help developers to improve the plugin. |
```

Comment:
```suggestion
| ballerina.enableTelemetry             | Enable telemetry logging. This will send log data to the extension developers. This will help developers to improve the extension. |
```